src/openapi/streaming/subscription.js

```diff
@@ -263,6 +290,32 @@ function onUnsubscribeError(referenceId, response) {
 }
 
 /**
+ * Called after modify patch is successful
+ * @param referenceId
+ * @param response
+ */
+function onModifyPatchSuccess(referenceId, response) {
+    if (referenceId !== this.referenceId) {
+        log.debug(LOG_AREA, 'Received an error response for patching a subscription');

```

Comment:
this is not an error response..
I think its just to return because it means the subscription has reset or been unsubscribed since the patch occurred..